.drone.yml

```diff
@@ -27,17 +40,16 @@ steps:
     depends_on: [deps-frontend]
 
   - name: lint-backend
-    pull: always
```

Comment:
Why remove this?

Reply:
`pull: always` is only necessary on the first occurence of an image, subsequent pulls do not result in any new data, so they are pointless.

Reply:
Actually, this needs to be added again because it is the first occurence. Will fix.